universal-login-react/src/core/utils/getSuggestion.ts

```diff
@@ -44,3 +46,9 @@ const isTakenOrInvalid = (actions: WalletSuggestionAction[], creations: string[]
     actions.includes(WalletSuggestionAction.create) && actions.length === 1;
   return creations.length === 0 && isOnlyCreateAction && source.length > 0;
 };
+
+const isInvalidForSuggestion = (actions: WalletSuggestionAction[], connections: string[], source: string) => {
+  const isOnlyConnectAction =
+    actions.includes(WalletSuggestionAction.connect) && !actions.includes(WalletSuggestionAction.create);
```

Comment:
Probably it will be more safe if we someday decide add new WalletSuggestionAction: 
```
actions.includes(WalletSuggestionAction.create) && actions.length === 1;
```